Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -1265,6 +1265,9 @@
   description: uplink-syndicate-pai-desc
   icon: { sprite: /Textures/Objects/Fun/pai.rsi, state: syndicate-icon-pai-off }
   productEntity: SyndicatePersonalAI
+  discountCategory: usualDiscounts  # Begin DeltaV
+  discountDownTo:
+    Telecrystal: 0 # End DeltaV
```

Comment:
just add 1 line comments on the first 2 lines
if you were going to have begin and theyd be separate lines